app/components/sidebar/SideBar.tsx

```diff
@@ -27,7 +28,19 @@ const ThemedSidebar = styled(Sidebar)`
     .item.active,
     .item:hover {
         background-color: ${props => props.theme.sidebarHoverActiveColor} !important;
-        color: ${props => props.theme.sidebarTextColor} !important;
+        color: ${props => props.theme.sidebarHoverActiveTextColor} !important;
+    }
+    :after {
+        position: fixed;
+        content: '';
+        width: 0.5em;
+        height: 0.5em !important;
+        -webkit-transform: rotate(45deg);
```

Comment:
As I remember, the `styled-components` comes with the "Automatic vendor prefixing" feature [Link to the information from the official docs](https://styled-components.com/docs/basics)

So I tihnk, that the `-webkit-transform: rotate(45deg);` could be replaced with `transform: rotate(45deg);` 😉 

```suggestion
        transform: rotate(45deg);
```

Reply:
Good to know that. I'll remove this line as the code is already in L39.